### DISCUSSION TITLE
Will there be a Chakra UI implementation for Solidjs?

### DISCUSSION THREAD

**Discussion by thedanchez** [chakra-ui/chakra-ui#5746] [General]
[Solid](https://www.solidjs.com/) is a newer declarative and *reactive* JS library for creating user interfaces. It is heavily inspired by React and is effectively React without the VDOM.

Was curious if Chakra UI would ever consider making their same components for Solid?

**Comment by thefrana** [comment]
They are working on a new approach that also supports SolidJS here: https://github.com/chakra-ui/zag

**Comment by RingOfStorms** [comment]
Will bump this thread now that v3 just dropped. I would be super stoked if v3 got solidjs version in the future.

**Comment by segunadebayo** [comment]
With the architectural design of Chakra v3, it's technically possible to support Solid.js.

We don't have the manpower to maintain this, but it believe a combination of [Solid Styled Component](https://github.com/solidjs/solid-styled-components) and [Ark UI Solid](https://github.com/chakra-ui/ark), it can be done.
